R/automateDataPreparation.R

```diff
@@ -7,6 +7,22 @@ automateDataPreparation <- function(datList = NULL, inputList, path = NULL,
 						addLeadingZeros=FALSE, truncateSpaceChar = TRUE, newID = NULL, oldIDs = NULL,
             missing.rule = list(mvi = 0, mnr = 0, mci = NA, mbd = NA, mir = 0, mbi = 0), verbose=FALSE) {
 
+  checkmate::assert_list(datList, types = "data.frame", null.ok = TRUE)
+  checkmate::assert_list(inputList, types = "data.frame", null.ok = TRUE)
+  checkmate::assert_character(path, len = 1, null.ok = TRUE)
+  lapply(c(filedat, filesps), checkmate::assert_character, len = 1)
+  lapply(c(readSpss, checkData, mergeData, recodeData, recodeMnr,
+           aggregateData, scoreData, writeSpss, collapseMissings, suppressErr,
+           rename, recodedData, addLeadingZeros, truncateSpaceChar, verbose), checkmate::assert_logical, len = 1)
+  checkmate::assert_numeric(breaks, null.ok = TRUE)
+  checkmate::assert_numeric(nMbi, len = 1, lower = 1)
+  checkmate::assert_scalar(rotation.id, null.ok = TRUE)
```

Comment:
rotation.id must be 
Character vector of length 1 (documentation, passed on to function mnrCoding indicating the name of the rotation indicator (e.g. “booklet”) in the dataset.)